helm/values-prod.yaml

```diff
@@ -16,100 +16,15 @@ microservice-chart:
-      OTEL_TRACES_EXPORTER: otlp
-      OTEL_METRICS_EXPORTER: otlp
-      OTEL_LOGS_EXPORTER: none
-      OTEL_TRACES_SAMPLER: "always_on"
-      ECS_SERVICE_NAME: pagopa-ecommerce-transactions-service-blue
-      ECS_SERVICE_ENVIRONMENT: "prod"
-      NPG_URI: "https://localhost/npg"
-      NPG_READ_TIMEOUT: "10000"
-      NPG_CONNECTION_TIMEOUT: "10000"
-      TRANSACTION_DOCUMENT_TTL: "600"
-      NPG_CARDS_PSP_LIST: "" #TODO to be filled with psp id list that are expected to be present into psp api keys configuration
-      CHECKOUT_BASE_PATH: "https://checkout.pagopa.it"
-      ECOMMERCE_EVENT_VERSION: "V1"
-      SEND_PAYMENT_RESULT_FOR_TX_EXPIRED_ENABLED: "true"
-      SESSION_URL_BASEPATH: "https://checkout.pagopa.it"
-      SESSION_URL_OUTCOME_SUFFIX: "/ecommerce-fe/esito"
-      SESSION_URL_CANCEL_SUFFIX: "/ecommerce-fe/cancel"
-      SESSION_URL_NOTIFICATION_URL: "https://api.platform.pagopa.it/ecommerce/npg/notifications/v1/sessions/{orderId}/outcomes?sessionToken={sessionToken}"
-      JWT_NPG_NOTIFICATION_VALIDITY_TIME: "900"
-      CHECKOUT_REDIRECT_PSP_LIST: ""
-      NODE_FORWARDER_URL: "https://api.platform.pagopa.it/pagopa-node-forwarder/api/v1"
-    CHECKOUT_REDIRECT_URL_MAPPING: >
-      { PSP1:'http://localhost' }
-
-      NODE_FORWARDER_READ_TIMEOUT: "10000" NODE_FORWARDER_CONNECTION_TIMEOUT: "10000" CHECKOUT_REDIRECT_LOGO_MAPPING: >
-
-
-
-
-        { PSP1:'https://host/assets/logo.png' }
-    envSecret:
-      MONGO_PASSWORD: mongo-ecommerce-password
-      REDIS_PASSWORD: redis-ecommerce-password
-      NODO_CONNECTION_STRING: nodo-connection-string
-      PAYMENT_TRANSACTIONS_GATEWAY_API_KEY: payment-transactions-gateway-api-key
-      JWT_ECOMMERCE_SECRET: sessions-jwt-secret
-      PERSONAL_DATA_VAULT_API_KEY: personal-data-vault-api-key
-      OTEL_EXPORTER_OTLP_HEADERS: elastic-otel-token-header
-      ECOMMERCE_PAYMENT_METHODS_APY_KEY: payment-method-api-key
-      WALLET_API_KEY: wallet-api-key
-      NPG_API_KEY: npg-api-key
-      NPG_CARDS_PSP_KEYS: npg-cards-psp-keys
-      JWT_NPG_NOTIFICATION_SECRET: npg-notification-signing-key
-      NODE_FORWARDER_API_KEY: node-forwarder-api-key
+      envConfig:
+        NODO_ECOMMERCE_CLIENT_ID: ecomm-blue
+        TRANSACTION_EXPIRATION_QUEUE_NAME: pagopa-p-weu-ecommerce-transactions-expiration-queue-b
+        TRANSACTION_CLOSE_PAYMENT_RETRY_QUEUE_NAME: pagopa-p-weu-ecommerce-transactions-close-payment-retry-queue-b
+        TRANSACTION_CLOSE_PAYMENT_QUEUE_NAME: pagopa-p-weu-ecommerce-transactions-close-payment-queue-b
+        TRANSACTION_REFUND_QUEUE_NAME: pagopa-p-weu-ecommerce-transactions-refund-queue-b
+        TRANSACTION_NOTIFICATIONS_QUEUE_NAME: pagopa-p-weu-ecommerce-transaction-notifications-queue-b
+        OTEL_RESOURCE_ATTRIBUTES: "service.name=pagopa-ecommerce-transactions-service-blue,deployment.environment=prod"
+        ECS_SERVICE_NAME: pagopa-ecommerce-transactions-service-blue
```

Comment:
I think you removed a few too many environment variables :)

Reply:
Those are blue canary delivery parameters, I applied the same refactoring done for UAT env, leaving only the parameters that have to be override starting from the green instance